proposals/numeric-intptr.md

```diff
@@ -0,0 +1,275 @@
+
+### 11.8.3 Unary minus operator
+https://github.com/dotnet/csharpstandard/blob/draft-v7/standard/expressions.md#1183-unary-minus-operator
+
+The predefined unary minus operators are:
+
+- Integer negation:
+
+  ```csharp
+  ...
+  nint operator –(nint x);
+  ```  
+
+  The result is computed by ... TODO2
+
+### 11.7.14 Postfix increment and decrement operators
+
+Predefined `++` and `--` operators exist for the following types: `sbyte`, `byte`, `short`, `ushort`, `int`, `uint`, **`nint`, `nuint`**, `long`, `ulong`, `char`, `float`, `double`, `decimal`, and any enum type.
+
+### 11.7.19 Default value expressions
+
+In addition, a *default_value_expression* is a constant expression if the type is one of the following value types: `sbyte`, `byte`, `short`, `ushort`, `int`, `uint`, **`nint`, `nuint`**, `long`, `ulong`, `char`, `float`, `double`, `decimal`, `bool,` or any enumeration type.
+
+### 11.8.5 Bitwise complement operator
+https://github.com/dotnet/csharpstandard/blob/draft-v7/standard/expressions.md#1185-bitwise-complement-operator
+
+The predefined bitwise complement operators are:
+
+```csharp
+...
+nint operator ~(nint x);
+nuint operator ~(nuint x);
+```
+
+### 11.8.6 Prefix increment and decrement operators
+
+Predefined `++` and `--` operators exist for the following types: `sbyte`, `byte`, `short`, `ushort`, `int`, `uint`, **`nint`, `nuint`**, `long`, `ulong`, `char`, `float`, `double`, `decimal`, and any enum type.
+
+## 11.9 Arithmetic operators
+
+### 11.9.2 Multiplication operator
+
+The predefined multiplication operators are listed below. The operators all compute the product of `x` and `y`.
+
+- Integer multiplication:
+
+  ```csharp
+  ...
+  nint operator *(nint x, nint y);
+  nuint operator *(nuint x, nuint y);
+  ```
+
+### 11.9.3 Division operator
+
+The predefined division operators are listed below. The operators all compute the quotient of `x` and `y`.
+
+- Integer division:
+
+  ```csharp
+  ...
+  nint operator /(nint x, nint y);
+  nuint operator /(nuint x, nuint y);
+  ```
+
+### 11.9.4 Remainder operator
+
+The predefined remainder operators are listed below. The operators all compute the remainder of the division between `x` and `y`.
+
+- Integer remainder:
+
+  ```csharp
+  ...
+  nint operator %(nint x, nint y);
+  nuint operator %(nuint x, nuint y);
+  ```
+
+### 11.9.5 Addition operator
+
+- Integer addition:
+
+  ```csharp
+  ...
+  nint operator +(nint x, nint y);
+  nuint operator +(nuint x, nuint y);
+  ```
+
+### 11.9.6 Subtraction operator
+
+- Integer subtraction:
+
+  ```csharp
+  ...
+  nint operator –(nint x, nint y);
+  nuint operator –(nuint x, nuint y);
+  ```
+
+## 11.10 Shift operators
+
+The predefined shift operators are listed below.
+
+- Shift left:
+
+  ```csharp
+  ...
+  nint operator <<(nint x, nint count);
+  nuint operator <<(nuint x, nint count);
```

Comment:
Shouldn't `count` be `int` for consistency with the other shift operators?

Or is this including the relaxations from https://github.com/dotnet/csharplang/issues/4666 ?

Reply:
Good catch. Bad search&replace ;-)